### PR TITLE
Add all u-root commands to NiChrome

### PR DESCRIPTION
**Pull request by rminnich**
Now that we are all source based adding in all the commands
is painless.

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

File: usb/urootlist.go [hunk 1]
```diff
@@ -4,48 +4,87 @@ package main
 
 var (
 	cmdlist = []string{
+		"github.com/u-root/u-root/cmds/ansi",
```
Comment (hugelgupf) [inline usb/urootlist.go]: Why not just glob?

Comment (rminnich) [reply usb/urootlist.go]: I thought you did not have that working yet.

Comment (rminnich) [reply usb/urootlist.go]: it actually failed last time I tried it :-)

Comment (hugelgupf) [reply usb/urootlist.go]: Argh. Glob works with directories. These are go package names, which don't support globs.

**Pull request merged**